lib/letter_avatar/avatar_helper.rb

```diff
@@ -1,16 +1,18 @@
 # encoding: UTF-8
-module LetterAvatar::AvatarHelper
+module LetterAvatar
+  module AvatarHelper
 
-  def letter_avatar_for(name, size = 64)
-    LetterAvatar.generate(name, size)
-  end
+    def letter_avatar_for(name, size = 64)
+      LetterAvatar.generate(name, size)
+    end
 
-  def letter_avatar_url_for(avatar_path)
-    avatar_path.to_s.gsub(/public/,'')
-  end
+    def letter_avatar_url_for(avatar_path)
+      avatar_path.to_s.sub('public/','/')
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.